testing.py

```diff
@@ -0,0 +1,116 @@
+import os 
+import unittest
+
+from app import app
+
+class testCase(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        pass
+
+    @classmethod
+    def tearDownClass(cls):
+        pass
+
+    def setUp(self):
+        self.app = app.test_client()
+        self.app.testing = True
+
+    def tearDown(self):
+        pass
+    
+    # Page Status 
+    def test_home_status(self):
+        result = self.app.get('/')
+        self.assertEqual(result.status_code, 200)
+
+    def test_builds_status(self):
```

Comment:
the test name should say what you are testing - example`test_builds_should_redirect`